### PR TITLE
Added empty line beetween

### PR DESCRIPTION
**Pull request by AdreeUA**

### CODE REVIEW

**Comment by AdreeUA** [comment]
Added configuration file for empty line between groups

**Comment by constverum** [comment]
Hey Andrew!

This is an awesome idea! The only thing is that by default the “border” property should be in the “visual” group. Therefore, before this change will be merged, needs to think about whether there is a way to implement the possibility of combining options and thus get a config which wanted the user (for example: with the “border” property into the "box model" or "visual" group and with or without an empty line between groups).

Of course, we can create a file for each case, but in this way, the implementation will looks ugly and not scalable (and yes, in the context of a simple definition of fields order, it may sound pretty funny =).

**Comment by AdreeUA** [comment]
Immediately apologize for my English. How about this solution?

**Comment by dougalg** [comment]
@constverum any chance of merging this? I'd love to have it in my project, actually.

File: README.md [hunk 1]
```diff
@@ -71,6 +71,38 @@ yarn add --dev stylelint stylelint-config-rational-order
...
+
+### Optional secondary options
+
+#### `"borderInBoxModel": Boolean`
+
+Default border property belongs to the *visual section* `"borderInBoxModel": false`. If `true` border property belongs to the *box model section*.
+
+#### `"emptyLineBeetweenGroup": Boolean`
+
+This option adds an empty line between groups. Default `false`.
```
Comment (theKashey) [inline README.md]: but it should defaults `true`?!

**Comment by theKashey** [comment]
It would be good to have `border` in a separate group. It would just end this war.

**Comment by constverum** [comment]
Sorry for the delay!
Andrew, you did a really awesome job, thank you! =)

This revision will be included in the next release with minor changes:
```
borderInBoxModel=> border-in-box-model
emptyLineBeetweenGroup => empty-line-between-groups
```

**Pull request merged**